plutus-use-cases/test/Spec/Lib.hs

```diff
@@ -25,3 +33,14 @@ reasonable (Ledger.ValidatorScript s) maxSize = do
 
 goldenPir :: FilePath -> CompiledCode a -> TestTree
 goldenPir path code = goldenVsString "PIR" path (pure $ fromString $ show $ pretty $ fromJust $ getPir code)
+
+staticFee :: Integer
+staticFee = 0
+
+timesFeeAdjust :: Integer -> Integer -> Value
+timesFeeAdjust multiplier change =
+    Ada.lovelaceValueOf $ change - staticFee * multiplier
```

Comment:
Can this be implemented in terms of `timesFeeAdjustV`? Also needs some comments explaining what it does